arches/app/views/etl_manager.py

```diff
@@ -37,18 +40,41 @@ def get(self, request):
         action = request.GET.get("action", None)
         loadid = request.GET.get("loadid", None)
         page = int(request.GET.get("page", 1))
+        filter_string = request.GET.get("filter", None)
         if action == "modules" or action is None:
             response = []
             for module in ETLModule.objects.all():
                 show = False if "show" in module.config.keys() and module.config["show"] is False else True
                 if self.request.user.has_perm("view_etlmodule", module) and show:
                     response.append(module)
         elif action == "loadEvent":
-            item_per_page = 5
-            all_events = LoadEvent.objects.all().order_by(("-load_start_time")).prefetch_related("user", "etl_module")
-            events = Paginator(all_events, item_per_page).page(page).object_list
-            total = len(all_events)
-            paginator, pages = get_paginator(request, all_events, total, page, item_per_page)
+            status_lookup = {
```

Comment:
I don't think this will work for a system that is not in English. The keys in here correspond to values that an admin could translate, and if that were the case, the filter would not work. The solution here is probably to use a dropdown in the UI.

Reply:
Even if the system is in English, there could be other 'status' types that exist in custom etl modules. I think we would need to get all the status types listed in the load_event status column to build the status dropdown.